src/configManager/models/database/sequelize/sequelizeConnector.js

```diff
@@ -50,10 +50,10 @@ async function getConfig() {
 async function getConfigValue(configValue) {
     const configClient = client.model('config');
     const options = {
-        attributes: { exclude: ['updated_at', 'created_at'] }
+        attributes: { exclude: ['updatedAt', 'createdAt'] }
     };
     options.where = { key: configValue };
-    let dbResult = await configClient.find(options);
+    let dbResult = await configClient.findOne(options);
```

Comment:
i would expect that findOne would throw exception if nothing found (also i think it's not the case) 
can you please verify it?
i also would expect it to return a single value and not array as line 57 assumes

Reply:
fixed